.changelogs/issue_190.yml

```
@@ -0,0 +1,7 @@
+significance: patch
+type: fixed
+links:
+  - "#190"
+attributions:
+  - "@seothemes"
```

```suggestion
attributions:
  - "@seothemes"
```
Attributions is for external contributors only :D